labelbox/schema/data_row.py

```diff
@@ -75,6 +73,9 @@ def update(self, **kwargs):
         # Convert row data to string if it is an object
         # All other updates pass through
         row_data = kwargs.get("row_data")
+        if not row_data and not kwargs.get("external_id") and not kwargs.get(
```

Comment:
I do not think you implemented the logic I mentioned here - https://github.com/Labelbox/labelbox-python/pull/1439#discussion_r1505196585.

What happens if `row_data` is set to a non-empty value and `global_key` is set to an empty value?

You might need to differentiate a field being unset from it being set but to an empty value.